src/flyte/types/__init__.py

```diff
@@ -34,3 +38,13 @@
     "guess_interface",
     "literal_string_repr",
 ]
+
+
+def _load_custom_type_transformers():
+    plugins = entry_points(group="flyte_type_transformers")
```

Comment:
for the cli plugins i used
```
entry_points(group="flyte.plugins.cli.commands")
```
and
```
entry_points(group="flyte.plugins.cli.hooks")
```

i think i'll change this to `flyte.plugins.types`?